rust/template/distributed_datalog/src/schema/relations.rs

```diff
@@ -0,0 +1,21 @@
+use std::collections::BTreeSet;
+
+use differential_datalog::program::RelId;
+
+/// A set of relations used in various contexts.
+pub type Relations = BTreeSet<RelId>;
```

Comment:
Since `RelId` may not be globally unique, what we might want instead is `BTreeSet<(Node, RelId)>`.

Reply:
Okay, that is interesting. In my world view the `RelId` *is* unique, as every node has the same view of the world based on the compiled program (which defines `RelId`s). Let's discuss that tomorrow.